index.bs

```diff
@@ -4,7 +4,8 @@ Shortname: nav-tracking-mitigations
 Repository: privacycg/nav-tracking-mitigations
 URL: https://privacycg.github.io/nav-tracking-mitigations/
 Editor: Pete Snyder, w3cid 109401, Brave https://brave.com/, psnyder@brave.com
-Editor: Jeffrey Yasskin, w3cid 72192, Google https://google.com/, jyasskin@google.com
```

Comment:
It should be possible to mark Jeffrey as a [former editor](https://speced.github.io/bikeshed/#metadata-former-editor).

Reply:
done. also added @wanderview as a former editor

Reply:
Thanks!